include/ArduinoJson/Print.hpp

```diff
@@ -7,6 +7,7 @@
 
 #pragma once
 
+#ifndef PARTICLE
```

Comment:
I looked into it. The latest stable particle firmware version 0.6.1 does define ARDUINO, and does not need the ifndef PARTICLE line in Print.hpp. Anything lower than 0.6.1 (including 0.6.0) won't compile without the ifndef PARTICLE in Print.hpp. I haven't tested uploading any code to a particle board yet, but it looks like the changes to Print.hpp may be unnecessary if compatibility is limited to the latest particle board firmwares.